src/languages/es.ts

```diff
@@ -1361,7 +1361,8 @@ export default {
         whereYouWork: '¿Dónde trabajas?',
         purpose: {
             title: '¿Qué quieres hacer hoy?',
-            error: 'Por favor, haga una selección antes de continuar.',
+            errorSelection: 'Por favor selecciona una opción para continuar.',
+            errorContinue: 'Por favor, haz click en continuar para configurar tu cuenta.',
```

Comment:
@suneox can you please follow the process for requesting translations in open source channel? include the english and Spanish copy in your post as well as screenshot of where this copy is used in the App.

Thank you!

Reply:
Sorry I did not know this was you/ did not connect the dots